SYCL/SpecConstants/2020/non_native/SpecConstBuffer.cpp

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: opencl-aot, cpu
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_x86_64 %S/Inputs/common.cpp -o %t.out \
+// RUN:          -fsycl-dead-args-optimization
+// RUN: env SYCL_PI_TRACE=-1 %CPU_RUN_PLACEHOLDER %t.out | FileCheck %s
+
+#include <cmath>
+#include <sycl/sycl.hpp>
+
+class Kernel1Name;
+
+const static sycl::specialization_id<int> SpecConst2{42};
```

Comment:
Nit: The numbering in `SpecConst2` seems a bit unnecessary as it is the only specialization constant in this test. Same goes for `Kernel1Name`.

Reply:
You're right, the numbering is unnecessary in this test.